yb-voyager/cmd/importData.go

```diff
@@ -933,17 +933,27 @@ func executeSqlFile(file string) {
 
 	var errOccured = 0
 	sqlStrArray := createSqlStrArray(file, "")
-	retryCount := INDEX_RETRY_COUNT
-	for i := 0; i < len(sqlStrArray); {
-		sqlStr := sqlStrArray[i]
-		log.Infof("Run query %q on target %q", sqlStr[1], target.Host)
+	for _, sqlStr := range sqlStrArray {
+		executeSqlStmtWithRetries(conn, sqlStr, &errOccured)
+	}
+
+	utils.WaitChannel <- errOccured
+	<-utils.WaitChannel
+}
+
+func executeSqlStmtWithRetries(conn *pgx.Conn, sqlStr []string, errOccured *int) {
```

Comment:
Instead of taking `errOccured` as an `out` parameter, why not just return `error` and let the caller compute errOccurred based on `err`?

Reply:
@amit-yb did this change. Please have a look